<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientInfoWIndow</class>
 <widget class="QWidget" name="PatientInfoWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatientInfoWindow</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>151</width>
     <height>701</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>113</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gender_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tel_lineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>151</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="gender_comboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>125</width>
      <height>36</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="tel_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TEL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remark_lineEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>201</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="remark_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Remark:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>351</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="address_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Home Address:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="edit_mode_checkBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>70</y>
      <width>97</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Mode</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="birthdate_dateEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>201</width>
      <height>42</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/MM/dd</string>
    </property>
   </widget>
   <widget class="QLabel" name="tel_label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Birthdate:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
